src/clr-angular/progress/spinner/spinner.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, Input } from '@angular/core';
+import { isBooleanAttributeSet } from '../../utils/component/is-boolean-attribute-set';
+
+const SPINNER_BASE_CLASS = 'spinner';
+
+const SPINNER_INVERSE = 'spinner-inverse';
+const SPINNER_INLINE = 'spinner-inline';
+
+const SPINNER_SMALL_SIZE = 'spinner-sm';
+const SPINNER_MEDIUM_SIZE = 'spinner-md';
+
+@Component({
+  selector: 'clr-spinner',
+  template: `
+    <ng-content></ng-content>
+  `,
+  host: {
+    '[attr.class]': 'assignClass',
+    '[attr.aria-live]': 'setAriaLive',
+    '[attr.aria-busy]': 'true',
+  },
+})
+export class ClrSpinner {
+  // Style
+  @Input() inline: boolean;
+  @Input() inverse: boolean;
+
+  // Size
+  @Input() small: boolean;
```

Comment:
We need to address the case when users do `<clr-spinner small="false">...loading</clr-spinner>` which could be confusing for them.

Reply:
I had a conversation with @coryrylan & @gnomeontherun and we ended up with the `prop="false"` is true. 

Ya so basically no matter what you add between `" "` if it's a string is `true`. But if you pass it like a `boolean` it will be ok

```
<component [clrSmall]="isAlwaysFalse"></component>
```

And I think that the idea is to be used more like that 
```
<component clrSmall></component>
```

Without any value attached to them